selfupdate.go

```diff
@@ -3,48 +3,44 @@ package main
 import (
 	"bufio"
 	"fmt"
+	"log"
+	"os"
+
 	"github.com/blang/semver"
 	"github.com/lawrencegripper/azbrowse/version"
 	"github.com/rhysd/go-github-selfupdate/selfupdate"
-	"log"
-	"os"
 )
 
 func confirmAndSelfUpdate() {
 	latest, found, err := selfupdate.DetectLatest("lawrencegripper/azbrowse")
 	if err != nil {
-		log.Println("Error occurred while detecting version:", err)
-		return
+		log.Panicln("Error occurred while detecting version:", err)
 	}
 
 	v, err := semver.Parse(version.BuildDataVersion)
 	if err != nil {
-		log.Println(err.Error())
-		return
+		log.Panicln(err.Error())
 	}
 	if !found || latest.Version.LTE(v) {
-		log.Println("Current version is the latest")
-		return
+		log.Panicln("Current version is the latest")
```

Comment:
I think this cause a crash when you have the latest version installed and a check is run as, if the check finds you have latest this `panicln` would run

Reply:
Oops - trigger-happy. I'll revert this one!